Doc/library/pprint.rst

```diff
@@ -36,7 +36,8 @@ Functions
 ---------
 
 .. function:: pp(object, stream=None, indent=1, width=80, depth=None, *, \
-                     compact=False, sort_dicts=False, underscore_numbers=False)
+                 compact=False, sort_dicts=False, underscore_numbers=False, \
+                 block_style=False)
```

Comment:
These are keyword-only (after `*`), so we have the luxury of alphabetical sorting:
```suggestion
                 block_style=False, compact=False, sort_dicts=False, \
                 underscore_numbers=False)
```